### PR TITLE
fix: removed unused certificate generation files

### PR DESCRIPTION
**Pull request by fboucquez**
It reduces the certificate folder, making it lighter for dir-comparision future unit tests.

### CODE REVIEW

**Review by Jaguar0625 — CHANGES_REQUESTED** [review]
you only need these 5 files on the server:
ca.crt.pem
ca.pubkey.pem
node.crt.pem
node.full.crt.pem
node.key.pem

**Comment by fboucquez** [comment]
Thanks @Jaguar0625, less is more. Removing other unused files.

File: test/service/CertificateService.test.ts [hunk 1]
```diff
@@ -70,26 +70,9 @@ describe('CertificateService', () => {
...
-            'new_certs',
-            'node.cnf',
-            'node.crt.pem',
-            'node.csr.pem',
-            'node.full.crt.pem',
-            'node.key.pem',
-            'serial.dat',
-            'serial.dat.old',
-        ]);
+        expect(files).deep.eq(['ca.cert.pem', 'ca.pubkey.pem', 'metadata.yml', 'node.crt.pem', 'node.full.crt.pem', 'node.key.pem']);
```
Comment (Jaguar0625) [inline test/service/CertificateService.test.ts:73]: i assume metadata.yml was kept intentionally?

**Review by Jaguar0625 — APPROVED** [review]

File: test/service/CertificateService.test.ts (hunk 1 above)
Comment (fboucquez) [reply test/service/CertificateService.test.ts:73]: Yes, metadata is mine. It holds the used main and node public keys. It's not easy nodejs to read the pem files without a library.

**Pull request merged**